canonical/canonical.com
# feat: Build '/solutions/sovereign-cloud' page and modal form

## Done

- Build '/solutions/sovereign-cloud' page based on [copydoc](https://docs.google.com/document/d/1XvdgVN87yqI9xOtFBaK9Z_Au8PTeHlQICzlYrgUKxuQ/edit) and [design](https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/24.10-Canonical.com%2Fsolutions%2Fsovereign-cloud?node-id=7-1254&t=NWmCG5I6aETEhEtb-0)
- Build modal form based on [copydoc](https://docs.google.com/document/d/1GkR3cbZbe7-gbwADceV8CkZK1xWI55nx8rSmsCLyuio/edit)

## QA

- Go to https://canonical-com-1280.demos.haus/solutions/sovereign-cloud
- Check the content matches that in the [copydoc](https://docs.google.com/document/d/1XvdgVN87yqI9xOtFBaK9Z_Au8PTeHlQICzlYrgUKxuQ/edit)
- Check on all screen sizes
- Check the modal opens and that the content matches the [copydoc](https://docs.google.com/document/d/1GkR3cbZbe7-gbwADceV8CkZK1xWI55nx8rSmsCLyuio/edit)
- Check the modal forms submits properly

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12009
